kubernetes/30-kyverno-setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# Update below if you have a different config.json you want to use.
+DOCKER_CONFIG_JSON=$HOME/.docker/config.json
+
+# Kyverno setup from the getting started tutorial:
+#   https://nirmata.com/2021/08/12/kubernetes-supply-chain-policy-management-with-cosign-and-kyverno/
+#   Installation: https://kyverno.io/docs/installation/
+
+# Create secrets for kaniko and kubernetes image pull
+
+# TODO: This should jsut be the normal secret if the kaniko task is updated to correctly use the docker config secret instead of requiring it to be hardcoded as config.json
+kubectl create secret generic secret-dockerconfigjson --type=opaque --from-file=config.json=$DOCKER_CONFIG_JSON --dry-run=client -o yaml | kubectl apply -f -
+
+# NOTE: Pull secret needs to exist in both kyverno namespace as well as the tekton task namespace (default in this case)
+kubectl create secret generic regcred --type=kubernetes.io/dockerconfigjson --from-file=.dockerconfigjson=$DOCKER_CONFIG_JSON -n kyverno  --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
It looks like the `kiverno` namespace needs to be created first:

```bash
❯ ./30-kyverno-setup.sh
+ DOCKER_CONFIG_JSON=/Users/rgreinhofer/.docker/config.json
+ kubectl create secret generic secret-dockerconfigjson --type=opaque --from-file=config.json=/Users/rgreinhofer/.docker/config.json --dry-run=client -o yaml
+ kubectl apply -f -
secret/secret-dockerconfigjson created
+ kubectl create secret generic regcred --type=kubernetes.io/dockerconfigjson --from-file=.dockerconfigjson=/Users/rgreinhofer/.docker/config.json -n kyverno --dry-run=client -o yaml
+ kubectl apply -f -
Error from server (NotFound): error when creating "STDIN": namespaces "kyverno" not found
```

Reply:
Oops! Also I need to make this change myself as my minikube just died